### ISSUE TITLE
COMEBin comparison

### ISSUE THREAD

**Issue by jsgounot** [RasmussenLab/vamb#286]
COMEBin, a new binning method [has recently been published in Nature com](https://www.nature.com/articles/s41467-023-44290-z#MOESM1). In the paper, Vamb performs relatively poorly compared to other binners in multiple situations, even compared to old ones such as MetaBat2. As Vamb is currently my default binner, I was curious to know what is your opinion on this matter, if you keep recommending using Vamb or Avamb, or if you see new software as better approaches.

**Comment by jakobnissen** [comment]
Dear @jsgounot 

We're looking into COMEBin now and trying to reproduce its results (but it might take some time as we are currently busy with other things).

Without speaking to COMEBin in particular, our general impression has been
* Vamb was pretty good when it came out, but the original Vamb is now worse than SemiBin2 (and MetaDecoder, IIRC)
* Vamb has improved since it came out, including Vamb's master branch being better than the released v.4.1.3
* SemiBin2 is better than Vamb, because it uses a single-copy genes-based reclustering, which is efficient for bacteria (and bacteria only). It's even better than Vamb on the master branch
* When applying SemiBin2's clustering to Vamb, Vamb becomes better
* We have an unreleased semi-supervised Vamb in the pipeline which is better still

So, of the currently released versions, SemiBin2 beats Vamb. The upcoming Vamb will beat SemiBin2.

**Comment by simonrasmu** [comment]
Chiming in here regarding single-copy genes. When these are used in the model (Metadecoder) or in the clustering (SemiBin1/2, COMEBin) and afterwards used to estimate how good your model is (CheckM) then it is a circular loop. Ie. you are optimising your model using SCGs and then afterwards using SCGs to check how good you are.

**Comment by jsgounot** [comment]
OK, thank you both for your honest answers. I know it's difficult to say, but do you have an estimate of version 5 release time?

**Comment by jakobnissen** [comment]
Not really. Almost surely this year. We're working on a manuscript but as you know, these things are unpredictable

**Comment by jsgounot** [comment]
Thanks for the update @jakobnissen, really appreciate that.
